packages/admin-portal/src/components/pages/devs-management/devs-management.tsx

```diff
@@ -181,49 +180,38 @@ export const DevsManagement: React.FC = () => {
     },
   ]
 
-  if (!isLoading && data?.length === 0) {
-    return (
-      <React.Fragment>
-        <Helper variant="info">
-          Unfortunately, there are no results that match your search criteria, please try again
-        </Helper>
-        <Link className="text-center" to={Routes.DEV_MANAGEMENT}>
-          <Button variant="primary" type="button">
-            New Search
-          </Button>
-        </Link>
-      </React.Fragment>
-    )
-  }
-
   return (
     <ErrorBoundary>
       <Section className="mb-0">
         <H3>Developer Management</H3>
       </Section>
       <DevsManagementFilterForm filterValues={filterValues} onSearch={onSearch} />
-      {isLoading || !data ? (
-        <Loader />
-      ) : (
-        <>
-          <Section>
-            <Table scrollable={true} loading={false} data={data || []} columns={columns} />
-          </Section>
-          <Section>
-            <div>Total: {totalCount}</div>
-          </Section>
-          <Pagination onChange={onPageChange} totalCount={totalCount} pageSize={pageSize} pageNumber={pageNumber} />
-          {/* <SetDeveloperStatusModal
+      {isLoading || !data ? <Loader /> : renderResult(data, columns, totalCount)}
+      <Pagination onChange={onPageChange} totalCount={totalCount} pageSize={pageSize} pageNumber={pageNumber} />
+      {/* <SetDeveloperStatusModal
         visible={isSetStatusModalOpen}
         afterClose={resetModal(false)}
         onSuccess={resetModal(true)}
         developer={developer}
       />  */}
-          <StatusModal visible={isSetStatusModalOpen} developer={developer} resetModal={resetModal} />
-        </>
-      )}
+      <StatusModal visible={isSetStatusModalOpen} developer={developer} resetModal={resetModal} />
     </ErrorBoundary>
   )
 }
 
+export const renderResult = (data, columns, totalCount) => {
+  return data?.length === 0 ? (
+    <Alert message="No Results " type="info" />
+  ) : (
+    <>
+      <Section>
```

Comment:
If you move Total to render content, pls check consistent between dev and apps management

Reply:
Sure, the apps management currently renders like below. I move the `total` to the top of the result to make it more consistent.
<img width="1167" alt="Screen Shot 2020-09-03 at 2 44 55 PM" src="https://user-images.githubusercontent.com/30467048/92086255-22cec200-edf4-11ea-85c6-5391b521467b.png">